app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+  end
+
+  def create
+    @payment = Payment.new(payment_params)
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
```

Comment:
I think this variable is only used when the purchase fails and we `render :new`, right?
If so, I would move it to the `else` block in line 22, so that it's clear it's only used there.